fdbserver/Knobs.cpp

```diff
@@ -443,6 +443,10 @@ ServerKnobs::ServerKnobs(bool randomize, ClientKnobs* clientKnobs, bool isSimula
 	init( STORAGE_DURABILITY_LAG_HARD_MAX,                    2000e6 ); if( smallStorageTarget ) STORAGE_DURABILITY_LAG_HARD_MAX = 100e6;
 	init( STORAGE_DURABILITY_LAG_SOFT_MAX,                     200e6 ); if( smallStorageTarget ) STORAGE_DURABILITY_LAG_SOFT_MAX = 10e6;
 
+	//FIXME: Low priority reads are disabled by assigning very high knob values, reduce knobs for 7.0
```

Comment:
Does that mean the perf test uses the high value, which may not trigger this feature in the test?

Reply:
we perf tested with the higher knob to confirm it does not change existing behavior, and perf tested with the knob set to see that it improved the result

Reply:
Is the knob in smallStorageTarget the knob used in your 2nd perf test?

Reply:
no, we tested with with a max lag of 100M versions and 275M versions (basically the intention is that for the real world we would go from e8 to e6 for both knobs

Reply:
the smaleStorageTarget is lower to ensure simulation tests the behavior

Reply:
I see.
I feel it would be valuable for SREs to know what knob value has been tested, which give them a good hint on what value they can use. 

Would you mind adding a comment on which value is used in your perf test that shows good performance? Thanks!